lambda/es-cleanup.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
```

Comment:
Add link of attribution to original lambda

Reply:
Also, mention copyrights and license

Reply:
Would we rather that in the function itself rather than https://github.com/cloudposse/terraform-aws-elasticsearch-cleanup/blob/0cb550ade3c70a52f7150ec4b7476ad2399772d4/README.yaml#L47-L48 ? NP if so